src/main/java/com/sweetwith/dailynote/domain/like/PostLike.java

```diff
@@ -22,7 +22,7 @@
     private Post post;
 
     @ManyToOne
-    //@JoinColumn(name="User_Id")
+    @JoinColumn(name="User_Id")
```

Comment:
I think id's column is Id as declared variable in User class due to not specifying.

https://www.icatpark.com/entry/JPA-%EA%B8%B0%EB%B3%B8-Annotation-%EC%A0%95%EB%A6%AC
https://wedul.site/507